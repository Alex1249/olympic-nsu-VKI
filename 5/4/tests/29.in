15
q k b e j i p r y t d v f a n
