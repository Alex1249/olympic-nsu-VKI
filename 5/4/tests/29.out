q k b e j i p r y t d v f n a
